### ISSUE TITLE
Am I use node-stl with react ?

### ISSUE THREAD

**Issue by mmterkc** [johannesboyne/node-stl#29]
Hi, I am new at react . I want to create a website with Django and react then I want to calculate stl files properties on client-side. Am I using node-stl package with react ? I use backend Django.

**Comment by connorlindsey** [comment]
No, this can only be used in a node environment. See this issue: https://github.com/johannesboyne/node-stl/issues/21
